sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/TerrainFactory.cs

```diff
@@ -99,6 +99,12 @@ internal override void OnInitialized()
 			_newTriangleList = new List<int>();
 		}
 
+		// Had to add this as we need a way of getting what was set in the Inspector so we can calculate real-life altitude.
+		public float GetHeightModifier()
```

Comment:
In this case, it makes sense to make the height modifier public, for changing at runtime.